src/lib/jsonParseV2/parseCompoundCommon.h

```diff
@@ -23,24 +23,32 @@
 * For those usages not covered by this license please contact with
 * iot_support at tid dot es
 *
-* Author: Ken Zangelin
+* Author: Fermín Galán
```

Comment:
GitHub shows this as a rename but actually the old file was deleted and the new one created.

NTC (informative)